q2_dada2/_denoise.py

```diff
@@ -350,13 +372,29 @@ def denoise_ccs(demultiplexed_seqs: SingleLanePerSampleSingleEndFastqDirFmt,
         for fp in nop_fp, filt_fp:
             os.mkdir(fp)
 
-        cmd = ['run_dada_ccs.R',
-               str(demultiplexed_seqs), biom_fp, track_fp, nop_fp, filt_fp,
-               str(front), str(adapter), str(max_mismatch), str(indels),
-               str(trunc_len), str(trim_left), str(max_ee), str(trunc_q),
-               str(min_len), str(max_len), str(pooling_method),
-               str(chimera_method), str(min_fold_parent_over_abundance),
-               str(n_threads), str(n_reads_learn)]
+        cmd = ['run_dada_single_paired_ccs.R',
+               '--input_directory', str(demultiplexed_seqs),
+               '--output_path', biom_fp,
+               '--output_track', track_fp,
+               '--removed_primer_directory', nop_fp,
+               '--filtered_directory', filt_fp,
+               '--forward_primer', str(front),
+               '--reverse_primer', str(adapter),
+               '--max_mismatch', str(max_mismatch),
+               '--indels', str(indels),
+               '--tuncation_length', str(trunc_len),
+               '--trim_left', str(trim_left),
+               '--max_expected_errors', str(max_ee),
+               '--tuncation_quality_score', str(trunc_q),
+               '--min_length', str(min_len),
+               '--max_length', str(max_len),
+               '--pooling_method', str(pooling_method),
+               '--chimera_method', str(chimera_method),
+               '--min_parental_fold', str(min_fold_parent_over_abundance),
+               '--num_threads', str(n_threads),
+               '--learn_min_reads', str(n_reads_learn),
+               '--homopolymer_gap_penalty', str('NULL'),
```

Comment:
```suggestion
               '--homopolymer_gap_penalty', 'NULL',
```

Reply:
String coercion of a string ('NULL') isn't necessary - the single quotes around 'NULL' will do the trick!